hawkular-alerts-rest/hawkular-alerts-rest-api/src/main/java/org/hawkular/alerts/rest/ActionPluginHandler.java

```diff
@@ -70,18 +70,17 @@ public ActionPluginHandler() {
             response = String.class, responseContainer = "List")
     @ApiResponses(value = {
             @ApiResponse(code = 200, message = "Successfully fetched list of actions plugins."),
+            @ApiResponse(code = 400, message = "Bad Request/Invalid Parameters.", response = ApiError.class),
```

Comment:
I think in this case perhaps a 400 is not necessary as this endpoint won't throw this one.
But no problem to maintain it as a pattern to not forget in other endpoints.